client/components/LearnerViewDirective.js

```diff
@@ -1025,12 +1038,21 @@ tie.directive('learnerView', [function() {
          * @param {string} code
          * @param {string} lang
          */
-        var storeCodeAndUpdateCachedCode = function(
-          questionId, code, lang) {
-          CodeStorageService.storeCode(questionId, code, lang);
+        var storeCodeAndUpdateCachedCode = function(questionId, code, lang) {
+          LocalStorageService.storeCode(questionId, code, lang);
           cachedCode = code;
         };
 
+        var storeFeedback = function() {
+          var latestFeedback =
+            $scope.feedbackStorage[$scope.feedbackStorage.length - 1];
+          LocalStorageService.storeFeedback(
```

Comment:
I actually think storeLatestFeedbackAndReinforcementBullets (or storeLatestFeedbackAndReinforcement) is fine. The names should be quite explicit about what's going on, and the requirements for what feedback we store may change in the future. Currently, it's reasonable for someone new reading the code to think that "feedback" refers to all the feedback since it's all right there in the browser window, hence the suggestion for "latestFeedback".

I agree that "feedbackAndReinforcement" is clumsy, but I think the correct fix for that is to figure out a name for that concept and use it throughout, rather than trying to paper over it here.